src/test/java/bio/terra/service/snapshotbuilder/query/LiteralTest.java

```diff
@@ -18,32 +18,32 @@
 class LiteralTest {
 
   @ParameterizedTest
-  @ArgumentsSource(QueryTestUtils.Contexts.class)
+  @ArgumentsSource(SqlRenderContextTest.Contexts.class)
   void renderString(SqlRenderContext context) {
     var literal = new Literal("foo");
     assertThat(literal.renderSQL(context), is("'foo'"));
   }
 
   @ParameterizedTest
-  @ArgumentsSource(QueryTestUtils.Contexts.class)
+  @ArgumentsSource(SqlRenderContextTest.Contexts.class)
   void renderStringEscaped(SqlRenderContext context) {
     var literal = new Literal("foo's");
     assertThat(literal.renderSQL(context), is("'foo’s'"));
   }
 
   @ParameterizedTest
-  @ArgumentsSource(QueryTestUtils.Contexts.class)
+  @ArgumentsSource(SqlRenderContextTest.Contexts.class)
   void renderInt(SqlRenderContext context) {
     var literal = new Literal(42);
     assertThat(literal.renderSQL(context), is("42"));
   }
 
   public static Stream<Arguments> renderBoolean() {
     return Stream.of(
-        Arguments.of(QueryTestUtils.createContext(CloudPlatform.GCP), true, "true"),
-        Arguments.of(QueryTestUtils.createContext(CloudPlatform.GCP), false, "false"),
-        Arguments.of(QueryTestUtils.createContext(CloudPlatform.AZURE), true, "1"),
-        Arguments.of(QueryTestUtils.createContext(CloudPlatform.AZURE), false, "0"));
+        Arguments.of(SqlRenderContextTest.createContext(CloudPlatform.GCP), true, "true"),
+        Arguments.of(SqlRenderContextTest.createContext(CloudPlatform.GCP), false, "false"),
+        Arguments.of(SqlRenderContextTest.createContext(CloudPlatform.AZURE), true, "1"),
+        Arguments.of(SqlRenderContextTest.createContext(CloudPlatform.AZURE), false, "0"));
```

Comment:
It seems strange that we're referencing another test here instead of a shared utiliy.

Reply:
My thinking was that the generator for a type should be inside the class that tests the type, rather than in a general utils class. It makes sense to me that if I'm looking for a test generator for `SqlRenderContext` I would look in `SqlRenderContextTest`.

Although I could try making the context generator a top level class of its own. Something like
```
public class SqlRenderContextProvider implements ArgumentsProvider {
  public static SqlRenderContext createContext(CloudPlatform cloudPlatform) {
    ...
  }
  public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
    ...
  }
}
```

Reply:
That's reasonable. I'm good with the generator living in the test.